spec/features/authentication_spec.rb

```diff
@@ -0,0 +1,73 @@
+feature 'authentication' do
+  it 'a user can log in if credentials are correct, and log in button dissapears' do
+    # Create a test user
+    user_create_dave
+
+    # Then log in as them
+    dave_log_in
+
+    expect(current_path).to eq '/'
+    expect(page).to have_content '@davedude'
+    expect(page).to_not have_link 'Log in'
+  end
+
+  it 'a user cannot log in if email is not registered, sees error at login screen' do
+    # Create a test user
+    user_create_dave
+
+    # Attempt log in with incorrect email
+    visit '/'
+    click_on 'Log in'
+    fill_in(:email, with: 'wrongemail@example.com')
+    fill_in(:password, with: 'password123')
```

Comment:
good use of test example - very clear and obvious that you are testing for a wrong email and I liked that this matches your readme gif presentation